src/sentry/receivers/sentry_apps.py

```diff
@@ -76,6 +79,33 @@ def send_comment_webhooks(organization, issue, user, event, data=None):
             data=data,
         )
 
+    from sentry.models.sentryfunction import SentryFunction
```

Comment:
@vpandiarajan20 I would recommend splitting out this change separately from the schema change in a different PR. Also, we should be checking the feature flag for the organization before doing anything. Lastly, we can just import `SentryFunction` from `sentry.models.`